internal/go_repository_test.go

```diff
@@ -113,15 +113,80 @@ func TestDirectives(t *testing.T) {
 	}
 }
 
-func TestRepoConfig(t *testing.T) {
-	if err := bazel_testing.RunBazel("build", "@bazel_gazelle_go_repository_config//:all"); err != nil {
-		t.Fatal(err)
-	}
+func TestRepoConfig(t *testing.T, m *testing.M) {
 	outputBase, err := getBazelOutputBase()
 	if err != nil {
 		t.Fatal(err)
 	}
 	outDir := filepath.Join(outputBase, "external/bazel_gazelle_go_repository_config")
+	testArgs := bazel_testing.Args{
```

Comment:
Any reason why we are creating a new `bazel_testing.Args` here rather than using the one defined at the top of this file?

Reply:
This was an accident: was trying to update the test configuration to address the config issue